src/sentry/runner/commands/cleanup.py

```diff
@@ -154,6 +155,35 @@ def cleanup(
     this can be done with the `--project` or `--organization` flags respectively,
     which accepts a project/organization ID or a string with the form `org/project` where both are slugs.
     """
+    # Tests already configure the app before calling _cleanup(),
+    # thus, we do it here to avoid conflicts when running tests
+    from sentry.runner import configure
+
+    configure()
+
+    _cleanup(
+        model=model,
+        days=days,
+        project=project,
+        organization=organization,
+        concurrency=concurrency,
+        silent=silent,
+        router=router,
+        timed=timed,
+    )
+
+
+def _cleanup(
```

Comment:
This cleanup function does not have any `click` boilerplate, thus, easier to use within tests.